packages/lib/apps/getUserTeamsQuery.ts

```diff
@@ -0,0 +1,53 @@
+import prisma from "@calcom/prisma";
+import { credentialForCalendarServiceSelect } from "@calcom/prisma/selects/credential";
+
+const getUserTeamsQuery = async (userId: number) => {
```

Comment:
Another helper function.

Reply:
Feels better suited to be in a repository

Reply:
Agreed. Let's use the repo directly. Wrapping into another function seems redundant.